config.go

```diff
@@ -62,6 +63,7 @@ type Config struct {
 	Table        string   `toml:"table"`
 	LogLevel     string   `toml:"log-level"`
 	Watch        bool     `toml:"watch"`
+	KubeConfig   string   `toml:"kubeconfig"`
```

Comment:
No the Go client for k8s is pretty clever, it will detect if it is running in a cluster and grab the appropriate info from there for you.  If you want to run this from outside the cluster you would need a kubeconfig to point to.